tests/integration/devfile/cmd_devfile_log_test.go

```diff
@@ -56,8 +57,7 @@ var _ = Describe("odo devfile log command tests", func() {
 
 			projectDir := filepath.Join(commonVar.Context, "projectDir")
 
-			helper.CmdShouldPass("git", "clone", "https://github.com/che-samples/web-nodejs-sample.git", projectDir)
-
+			helper.CopyExample(filepath.Join("source", "web-nodejs-sample"), projectDir)
```

Comment:
We can use the current nodejs sample directory here. It seems to have worked for me locally.

Reply:
Correct me if I'm wrong but I think things would work with many nodejs projects. Besides, I don't mind testing a different example in our tests as compared to testing only one example project - for example, I'd prefer we test something beyond nodejs+etcd operator for Operator Hub tests at some point.

Reply:
OK let's test against a new example